demo/App.js

```diff
@@ -28,11 +29,30 @@ class App extends Component {
         });
     }
 
+    renderMode() {
+        const mode = Environment.searchParams.get('mode');
+
+        if (mode === AppMode.Filtering) {
+            return (<button
+                className='clear-filters'
+                onClick={() => {
+                    const tableProps = R.clone(this.state.tableProps);
+                    tableProps.filtering_settings = '';
+
+                    this.setState({ tableProps });
+                }}
+            >Clear Filter</button>);
+        }
+    }
```

Comment:
For this new case, need an additional button to trigger the filter reset -- either that or make this a full on e2e test, which I think is pointless.

Reply:
Mode based conditional additional comps